<?xml version="1.0"?>

<launch>
    
    <node name="free_space_navigation_node" pkg="toy_robot" type="free_space_navigation_node" output="screen"/>
    
    <node name="input" pkg="toy_robot" type="input" output="screen" launch-prefix="xterm -e"  respawn="true"/>

    <test test-name="test_node" pkg="toy_robot" type="test_node"/>


</launch>
